### PR TITLE
Add circle CI

### PR DESCRIPTION
**Pull request by demiladebamgbose**
added file circle.yml and package.json files

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+machine:
+  node:
+    version: 6.5.0
+dependencies:
+  override:
+    - npm install --dev
```
Comment (rowlandekemezie) [inline circle.yml]: You don't need to `override` because CIRCLE CI would use `npm install` to handle all dependencies.

**Comment by rowlandekemezie** [comment]
@demiladebamgbose Add some test cases for your build on CI to pass

**Comment by demiladebamgbose** [comment]
I added one test. The  build now passes @andela-rekemezie

**Comment by rowlandekemezie** [comment]
@demiladebamgbose This issue is closed already. Take a look and respond the comments on your PR. #2 😄

**Pull request closed without merging**